sda/cmd/ingest/ingest.go

```diff
@@ -75,10 +76,9 @@ func main() {
 		panic(err)
 	}
 	app.ArchiveKeyList, err = config.GetC4GHprivateKeys()
-	if err != nil {
-		log.Error(err)
+	if err != nil || len(app.ArchiveKeyList) == 0 {
 		sigc <- syscall.SIGINT
-		panic(err)
+		panic(errors.New("no C4GH private keys configured"))
```

Comment:
If we're failing because of an error, it's helpful showing what.

Reply:
The errors are printed in the`GetC4GHprivateKeys` function

Reply:
It's seldom I feel there's too much information about why something fails and I would prefer not to rely on some other part of code to do things that aren't obviously part of the contract.

But if you feel it's enough it's approved.